### DISCUSSION TITLE
Filters have no effect when pulled in from Other options

### DISCUSSION THREAD

**Discussion by InkingHubris** [Xaymar/obs-stream-effects#924] [Q&A] [answered]
I may be missing something, but I am trying to automate a 3D transform. 

When I add the Move Value, choose the 3D Transform from the filter, move value type = settings and Get values, everything updates. I then go through the rest of my settings, duration, end delay etc. I then enable the start trigger and next move (Enable when eye icon is visible and reverse, respectively).

Yet, when I test, the eye icon turns on, but no transitions occur.

Using OBS 27.2.4 and latest stable (release) version of StreamFX

**Comment by Xaymar** [comment, marked as answer]
This is probably a Move Transition bug, as StreamFX has no such functionality or limitation. If you tell a StreamFX feature to change something, they will respond as soon as the next video tick occurs.

**Comment by InkingHubris** [comment]
I will look into that, then. Thanks.
